src/App.js

```diff
@@ -1,24 +1,25 @@
 import axios from "axios";
 import { useState } from "react";
 import { useEffect } from "react";
-import "./App.css";
+import { Footer, Sidebar } from "./Components";
 
 function App() {
   const [someData, setSomeData] = useState([]);
```

Comment:
It will better if give better naming for your states , eg- products

Reply:
its just for testing purposes